index.bs

```diff
@@ -676,6 +670,17 @@ The initial value of ModuleStatus.prototype.constructor is the intrinsic object
 1. Return _stageEntry_.[[Stage]].
 </emu-alg>
 
+<h4 id="module-status-module">get ModuleStatus.prototype.key</h4>
```

Comment:
Let's call it `ModuleStatus.prototype.originalKey` since it can be deleted from the place it was originally stored in the registry of its original loader. It doesn't need to be a nice short name since it's a rarely used reflective property.